app/controllers/appeals_base_controller_v1.rb

```diff
@@ -9,6 +9,14 @@ class AppealsBaseControllerV1 < ApplicationController
 
   private
 
+  def log_non_module_controller(action:, form_id:)
```

Comment:
Can this method be thrown into a helper or something to reduce the repeated methods?

Reply:
I can do that. The method is slightly different for the form controllers that inherit from this base controller, and for the other two controllers. All this logging is temporary while we transition to our new controllers, so it doesn't feel as problematic to have some duplication. Does that sound ok?

Reply:
Clarification: is it ok not to make any changes since this code is only intended to exist for about 2 weeks?

Reply:
Yes, let's leave since it's temporary